sharing_portal/migrations/0018_auto_20220317_1314.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2 on 2022-03-17 18:14
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sharing_portal', '0017_artifact_trovi_uuid'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='daypassrequest',
+            name='artifact',
+        ),
+        migrations.AddField(
+            model_name='daypassrequest',
+            name='artifact_uuid',
+            field=models.CharField(max_length=36, null=True),
+        ),
+        migrations.CreateModel(
```

Comment:
How will we handle existing requests (and does it matter?) Normally you would do a dance like, add the new column, then do a data migration, then remove the old column.